File: apcs/cpp/zj-e286.cpp

#include <iostream>
using namespace std;

int main() {
    int a[4] = {0}, n;
    for (int i = 0; i < 4; i++){
        for (int j = 0; j < 4; j++){
            cin >> n;
            a[i] += n;
        }
    }
    cout << a[0] << ":" << a[1] << "\n";
    cout << a[2] << ":" << a[3] << "\n";
    if (a[0] > a[1] && a[2] > a[3]) cout << "Win\n";
    else if (a[0] < a[1] && a[2] < a[3]) cout << "Lose\n";
    else cout << "Tie\n";
}

File: apcs/python/zj-e286.py
#!/usr/bin/env python 

a = []
for i in range(4):
    x = list(map(int, input().split()))
    a.append(sum(x))

print(f"{a[0]}:{a[1]}")
print(f"{a[2]}:{a[3]}")

if (a[0] > a[1] and a[2] > a[3]):
    print("Win")
elif (a[0] < a[1] and a[2] < a[3]):
    print("Lose")
else:
    print("Tie")
